src/vtt-segment-loader.js

```diff
@@ -235,6 +203,25 @@ export default class SegmentLoader extends videojs.EventTarget {
   }
 
   /**
+   * Indicates which time ranges are buffered
+   */
+  buffered() {
+    if (!this.subtitlesTrack_ || !this.subtitlesTrack_.cues.length) {
+      return videojs.createTimeRanges();
+    }
+
+    const cues = this.subtitlesTrack_.cues;
+    let start = cues[0].startTime;
+    let end = cues[cues.length - 1].startTime;
+
+    return videojs.createTimeRanges([[start, end]]);
+  }
+
+  timestampOffset() {
+    return this.syncController_.timestampOffsetForTimeline(this.currentTimeline_);
```

Comment:
from #1043 
@imbcmdth: Is this function used anywhere? If so, it needs a comment.